template-sample-app/template/src/components/views/datasets/Datasets.js

```diff
@@ -55,6 +55,8 @@ function Datasets() {
     };
   }, [credentials, dispatch]);
 
+  // Auto import useEffect
+
   return (
     <Grid
```

Comment:
We're gonna remove this file (all related to that tab), but I'll wait till this gets merged, to avoid any conflict